src/Cart.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { deleteOrderProductsThunk, updateOrderProductThunk, se } from './store';
+import { setOrderProductsThunk, setOrdersThunk, setProductsThunk, setUsersThunk} from './store'
+
+
+
+class _Cart extends React.Component {
+  constructor(props) {
+    super();
+    this.deleteItem = this.deleteItem.bind(this);
+    this.updateItem = this.updateItem.bind(this);
+  }
+  // async componentDidMount() {
+  //   await this.props.getUsers()
+  //   await this.props.getProducts()
+  //   await this.props.getOrders()
+  //   await this.props.getOrderProdcuts()
+  // }
+  async deleteItem (id){
+    await this.props.deleteItem(id)
```

Comment:
If you're using thunks you should never be `await`ing in a React component.

Reply:
got rid of async and await